make/STM32F4xx/common.mk

```diff
@@ -95,6 +95,10 @@ ifeq (STM32F446xC, $(DEVICE))
     MAPPED_DEVICE = STM32F446xE
```

Comment:
Change this line to map to STM32F446xx too. Turns out both the STM32F446xC and STM32F446xE need to be mapped to STM32F446xx.